thirdparty/detray/CMakeLists.txt

```diff
@@ -30,11 +30,21 @@ set(DETRAY_CUSTOM_SCALARTYPE
     "Scalar type to use in the Detray code"
 )
 
-set(DETRAY_BUILD_TESTING
+set(DETRAY_BUILD_TEST_UTILS
+    ON
+    CACHE BOOL
+    "Turn on the build of the Detray test utilities"
+)
+set(DETRAY_BUILD_UNITTESTS
     OFF
     CACHE BOOL
     "Turn off the build of the Detray unit tests"
 )
+set(DETRAY_BUILD_INTEGRATIONTESTS
+    OFF
+    CACHE BOOL
+    "Turn off the build of the Detray integration tests"
+)
```

Comment:
I would also switch off the benchmarks here (```DETRAY_BUILD_BENCHMARKS```)

Reply:
Done.